letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -560,7 +560,7 @@ BootstrapMageiaCommon() {
       exit 1
   fi
 
-  if ! $SUDO urpmi --force \
+  if ! $SUDO urpmi --force $QUIET_FLAG \
```

Comment:
This doesn't really matter to me either.  Not having to open another PR saves me some typing and clicking a few more buttons :smile: